Umbraco-Heartcore/Getting-Started-Cloud/The-Cloud-Portal/index.md

```diff
@@ -1,74 +1,74 @@
 # The Umbraco Cloud Portal
 
-In this article you will learn more about the Umbraco Cloud Portal and what options that are available to you.
+In this article, you will learn more about the Umbraco Cloud Portal and what options that are available to you.
 
 ## Video tour of the Portal
 
 <iframe width="800" height="450" src="https://www.youtube.com/embed/-wARG4wyk2c?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 
 ## Written tour of the Portal
 
-After you have logged into the [Umbraco Cloud Portal](https://www.umbraco.io) you are presented with the project overview.
+After you have logged into the [Umbraco Cloud Portal](https://www.umbraco.io), you are presented with the project list overview.
 
 ![Portal Project Overview](images/portalOverview.png)
 
-From here you can see a list of all your projects. If you have many projects, you are able to search for specific projects from the search bar in the top right corner.
+From here you can see a list of all your projects. If you have several projects, you can search for specific projects by using the search bar in the top right corner.
 
-Clicking on a project will take you to the project where you will get an overview of your environment(s).
+Clicking on a project will take you to the project page where you will get an overview of your environment(s).
 
 ![Portal Project Overview](images/projectOverview.png)
 
 ## Manage Environments
 
-From here you are able to create multiple environments so you might have a Development environment as well. Depending on what plan you have chosen, you will have access to one or more environment on your project.
+From here, you are able to create multiple environments, so that you can have a Development and eventually also a Staging environment.. Depending on what plan you have chosen, you will have access to one or more environment on your project.
```

Comment:
```suggestion
From here, you are able to create multiple environments, so that you can have a Development and eventually also a Staging environment. Depending on what plan you have chosen, you will have access to one or more environments on your project.
```